cliquet/tests/test_resource.py

```diff
@@ -0,0 +1,327 @@
+from mock import sentinel, MagicMock, patch
```

Comment:
Maybe a matter of taste, but I would keep them as ``mock.sentinel``, ``mock.MagicMock``, ``mock.patch``